common/src/main/java/com/wynntils/core/text/PartStyle.java

```diff
@@ -68,13 +68,6 @@ static PartStyle fromStyle(Style style, StyledTextPart owner, Style parentStyle)
         } else {
             // This changes properties that are null, as-in, inherting from the previous style.
             inheritedStyle = style.applyTo(parentStyle);
-
-            // We don't want to inherit these properties.
-            inheritedStyle = inheritedStyle
-                    .withClickEvent(style.getClickEvent())
-                    .withHoverEvent(style.getHoverEvent())
-                    .withInsertion(style.getInsertion())
-                    .withFont(style.getFont());
```

Comment:
I was too fast to read the code of Style when I first wrote this.. so I made the mistake of not inheriting this.